Atomic/syscontainers.py

```diff
@@ -407,13 +407,12 @@ def _do_checkout(self, repo, name, img, upgrade, values, destination, unitfileou
                     pass
             rootfs = os.path.join(destination, "rootfs")
 
-        if os.path.exists(destination):
-            shutil.rmtree(destination)
-
```

Comment:
this was used only with `upgrade`.  `install` would fail earlier if the destination is not empty:

```
        if self.get_checkout(name):
            util.write_out("%s already present" % (name))
            return
```